packages/modules/electricity_tariffs/voltego/tariff.py

```diff
@@ -49,12 +58,12 @@ def fetch(config: VoltegoTariff) -> None:
         timezone = "UTC+2:00"
     else:
         timezone = "UTC+1:00"
-    raw_prices = req.get_http_session().get(
-        "https://api.voltego.de/market_data/day_ahead/DE_LU/60",
-        headers={"Content-Type": "application/json;charset=UTF-8",
-                 "Authorization": f'Bearer {config.configuration.token.access_token}'},
-        params={"from": start_date, "tz": timezone}
-    ).json()["elements"]
+    # Bei Voltego wird anscheinend nicht ein Token pro Client, sondern das letzte erzeugte gespeichert.
+    try:
+        raw_prices = get_raw_prices()
+    except requests.exceptions.HTTPError:
```

Comment:
Kann hier der HTTP Fehlercode weiter eingegrenzt werden? Ich vermute, dass ein 401 "Unauthorized" vom Server kommt. Bei anderen Stati, insbesondere 5xx macht es ja keinen Sinn, dann nochmal eine Anfrage zu schicken. Die wird dann vermutlich ebenfalls fehlschlagen.
```suggestion
    except HTTPError as error:
        if errore.response.status_code != 401:
            raise error
```
(ungetestet!)